.github/workflows/build.yml

```diff
@@ -353,6 +353,12 @@ jobs:
           path: dist
       - name: Generate checksum file
         run: cd dist && sha256sum * > "k6-${VERSION}-checksums.txt"
+      - name: Anchore SBOM Action
```

Comment:
I'm wondering if we should block the release if this action fails :thinking: Like should this part has something like continue-on-error https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

@tungbq @mstoykov what do you think?

Reply:
@olegbespalov Yep, I understand the concern as this is part of the release process, and this is the first time we integrate it.
From my side, I'd prefer adding `continue-on-error:true` too.
Not sure how important this SBOM file is (from the release perspetive), but I think we can first add `continue-on-error:true` flag, then monitor some releases and once everything stable, we can disable this flag in the future.

Reply:
I think this is fine for now.

Preferably we will get notified if this doesn't run, but to be honest I prefer to get this silently broken then having to redo a release because of it.

I guess if we can make it a step *after* the release that just uploads it :thinking: ... that might work :shrug:

Reply:
Okay, since the concern is really hypothetical let's leave it for now :+1:

Reply:
I think my comment was not clear - I am fine with what @olegbespalov proposes.

`continue-on-error:true` will also not send email to the person who triggered the workflow if it fails. So it will "silently" fail, which was what I referred to.

But breaking the whole release for this seems ... bad to me